cristin-project/src/main/java/no/unit/nva/cristin/projects/query/QueryCristinProjectHandler.java

```diff
@@ -2,6 +2,7 @@
 
```

Comment:
I like the code you have written in this class. Very creative and reduces a lot of one by one parameter logic 🙂 You can also consider moving the logic shared between this handler class and the other handler class to avoid duplication. We can discuss that later, as im not sure where we can put it, but my first thought is either their superclass and/or cristin-commons module's util class